tests/test_cli.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+from pytest import fixture
+from subprocess import Popen, PIPE
+from os import getcwd, X_OK, access
+from pathlib import Path
+
+
+class CLITestDefaultValues:
+    GRINDER_RUN_COMMAND = "./grinder.py"
+    CLI_QUERY_PATH = "tests/test_data/test_queries/cli_arguments_test.json"
+    if Path(getcwd()).stem == "tests":
+        GRINDER_RUN_COMMAND = f".{GRINDER_RUN_COMMAND}"
+        CLI_QUERY_PATH = CLI_QUERY_PATH[len(Path(getcwd()).stem) + 1 :]
+    if not access("grinder.py", X_OK) and not access("../grinder.py", X_OK):
+        if Path(getcwd()).stem == "tests":
+            GRINDER_RUN_COMMAND = f"python3 {GRINDER_RUN_COMMAND}"
+        else:
+            GRINDER_RUN_COMMAND = f"python3 {GRINDER_RUN_COMMAND[2:]}"
+
+
+@fixture
+def cli_args(shodan_key: str, censys_id: str, censys_secret: str) -> list:
+    """
+    Sets all needed arguments for execution and testing
+    :param shodan_key: shodan key value
+    :param censys_id: censys id value
+    :param censys_secret: censys secret value
+    :return: all constant arguments
+    """
+    sk = shodan_key
+    ci = censys_id
+    cs = censys_secret
```

Comment:
I think that you don't really need these variables: `sk`, `ci`, `cs`.
You can put `shodan_key`, `censys_id`, `censys_secret` directly in your `base_args` command.